lib/index.ts

```diff
@@ -56,7 +56,7 @@ export class Oops extends Error {
 }
 
 export const getErrorCategory = (err): ErrorCategory => {
```

Comment:
should err be optinal parameter now?

Reply:
Err is any type, so it won't be checked on compile time anyway.